obspy/core/util/misc.py

```diff
@@ -639,6 +639,34 @@ def buffered_load_entry_point(dist, group, name):
     return _ENTRY_POINT_CACHE[hash_str]
 
 
+if sys.version_info[0] < 3:
```

Comment:
Please move it to this file as we already have a version of this function for the other way around: https://github.com/obspy/obspy/blob/master/obspy/core/compatibility.py#L99

Reply:
I dont see a test file for the compatibility module, and I cant find any direct tests for any of those functions. Should I create a new test module or leave the tests for py3_round in test_util_misc.py ?

Reply:
create a new test module + move the test - tests should be in the same module as the tested function